packages/ui-source-code-editor/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@instructure/ui-source-code-editor",
+  "version": "8.25.0",
+  "description": "A UI component library made by Instructure Inc.",
+  "author": "Instructure, Inc. Engineering and Product Design",
+  "module": "./es/index.js",
+  "main": "./lib/index.js",
+  "types": "./types/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/instructure/instructure-ui.git"
+  },
+  "homepage": "https://instructure.github.io/instructure-ui/",
+  "bugs": "https://github.com/instructure/instructure-ui/issues",
+  "scripts": {
+    "lint": "run -T ui-test --lint",
+    "lint:fix": "run -T ui-test --lint --fix",
+    "clean": "run -T ui-build --clean",
+    "build": "run -T ui-build --modules es,cjs",
+    "build:watch": "yarn ts:check --watch & run -T ui-build --watch",
+    "build:types": "run -T tsc -p tsconfig.build.json",
+    "ts:check": "run -T tsc -p tsconfig.build.json --noEmit --emitDeclarationOnly false"
+  },
+  "license": "MIT",
+  "devDependencies": {
+    "@instructure/ui-babel-preset": "8.25.0",
+    "@instructure/ui-test-queries": "8.25.0",
+    "@instructure/ui-test-utils": "8.25.0",
+    "@instructure/ui-themes": "8.25.0",
+    "@types/lodash": "^4"
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.13.10",
```

Comment:
oh I know this one, sorry for the false alert. Its some common Babel stuff that is supposed to decrease the final binary size because it will be included only once.